app/src/main/java/cm/aptoide/pt/ApplicationModule.java

```diff
@@ -1085,6 +1095,16 @@ AnalyticsBodyInterceptorV7 provideAnalyticsBodyInterceptorV7(
         .build();
   }
 
+  @Singleton @Provides @Named("retrofit-wallet") Retrofit providesV8SecondaryRetrofit(
+      @Named("v8") OkHttpClient httpClient, @Named("base-secondary-host") String baseHost,
+      Converter.Factory converterFactory, @Named("rx") CallAdapter.Factory rxCallAdapterFactory) {
+    return new Retrofit.Builder().baseUrl(baseHost)
```

Comment:
This base host is the v7 one or am I seeing it wrong? The only thing it changes is the user-agent (which is the v8 one)

Reply:
Also not related to the ticket scope but please extract the DONATIONS_URL to the build config. This should definitely not be here - also should be something like blockchain_base_url but we can do the renaming later because we need to refactor it to allow toolbox to configure the http or https part.